auction-deploy/auction_deploy/core.py

```diff
@@ -0,0 +1,132 @@
+import json
+import pkg_resources
+from typing import Dict, NamedTuple
+from collections import namedtuple
+
+from eth_keyfile import extract_key_from_keyfile
+from deploy_tools.deploy import send_function_call_transaction, deploy_compiled_contract
+
+
+class AuctionOptions(NamedTuple):
+    start_price: int
+    auction_duration: int
+    number_of_participants: int
+    release_block_number: int
+
+
+DeployedAuctionContracts = namedtuple("DeployedContracts", "locker slasher auction")
+
```

Comment:
Please change this to use a subclass of NamedTuple like it is done a few lines above for the AuctionOptions class (in order to keep it consistent in a single file).

Reply:
done